app/main.py

```diff
@@ -1,2 +1,55 @@
 class Dictionary:
-    pass
+    def __init__(self):
+        self.capacity = 8
+        self.load_factor = 0.66
```

Comment:
0.67 is a better approximation than 0.66. :)
You may also use `2 / 3`, I think it's even better.